src/components/ChecDataPill.vue

```diff
@@ -66,6 +66,14 @@ $colors: 'green', 'orange', 'purple', 'red', 'blue';
     }
   }
 
+  &--green {
+    @apply bg-green-100 text-green-700;
+
+    &.data-pill--filled {
+      @apply bg-green-600 text-white;
```

Comment:
Is there a reason we didn't just change the 600 and 500 values to work, rather than this? It feels a bit weird that green is the only colour with a 700 value now.

Reply:
I did this in the event we still want to use that color in other places. I'll update to remove the original and shift it down.

Reply:
@ScopeyNZ updated.